src/utils/getTokenList.ts

```diff
@@ -75,7 +75,15 @@ export default async function getTokenList(
     }
 
     const [json] = await Promise.all([response.json(), tokenListValidator])
-    if (BYPASS_LIST.indexOf(listUrl) >= 0) return json
+    const parsedData = !!json.data
```

Comment:
Should be `json?.data?.tokens ? ...`

Or `json?.data?.tokens?.length ? ...`

Reply:
totally agree, should check carefully to prevent crash